.github/workflows/ci.yaml

```diff
@@ -22,6 +22,11 @@ jobs:
         make ci
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Run helm-docs to check if chart README is staled
```

Comment:
I don't think we want to release the chart on CI due to how our images builds work. Eventually, I can see a separate CI action on a new tag that automatically cuts a release, but we will need to figure out how to include an image build into that workflow.

Reply:
Perhaps I'm misunderstanding this step, though.

Reply:
It's just checking if the readme is staled and needs updated. eg: values file changes, chart version changes, etc..

Reply:
Gotcha. Currently we generate a number of files using `make codegen`. This is run as part of `make verify` in the CI. CI will fail if there are any modified files in the workspace. If we include `helm-docs` in the `make codegen` step, then everything should wire up perfectly.

Reply:
Eventually we might want to break this apart into different GH actions, but for now, this is how we've unified this process of codegen and CI verifcation.